diff --git a/base/BUILD.gn b/base/BUILD.gn
index 37b8fc7c5f035ad9eeeda8738ee2e891faf1ae29..c8e3aa3bbcc3f9e2b498a8483c5e5a0755ce1adb 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4825,6 +4825,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5482,6 +5483,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
