diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index ae4e56df06dcdee85bbb5788d14c74c8914306a0..ec30dea539a86946079770c00d9fe36728e965d9 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -1101,6 +1101,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   }
 
   if (!url.SchemeIs(content::kChromeUIScheme)) {
+    if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
   }
 
