diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index 92b40136a473d777d19bc68d49892da8ee6fd343..fb75e2532ecd89b6749912640cfc54aa1989a1a8 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -134,8 +134,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(
   std::unique_ptr<TabHoverCardController> hover_card_controller(
       std::make_unique<TabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
-                                              std::move(hover_card_controller));
+  BRAVE_TAB_STRIP_REGION_VIEW_CREATE_TAB_STRIP
   return tab_strip;
 }
 
