diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index 593667735e0d2a7930bb273c25afa6e4d2d55661..985e9e3ee79274bb38a7a292a18b70f833de3042 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -139,8 +139,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(
   std::unique_ptr<TabHoverCardController> hover_card_controller(
       std::make_unique<TabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
-                                              std::move(hover_card_controller));
+  BRAVE_TAB_STRIP_REGION_VIEW_CREATE_TAB_STRIP
   return tab_strip;
 }
 
