diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 78dd0c83d3ed3a77ad4f7ddc667699cae4bc4343..c3093aea058766a07bfbced3db108e49a42562ac 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -212,6 +212,7 @@ class TabStripModel {
   TabStripModel(const TabStripModel&) = delete;
   TabStripModel& operator=(const TabStripModel&) = delete;
 
+  virtual
   ~TabStripModel();
 
   // Retrieves the TabStripModelDelegate associated with this TabStripModel.
