diff --git a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
index 6e666561fc360fa1d8f2749f035811c8cfae530c..9c2ec59e5454b85ed031feb4ff6dd5058670ab53 100644
--- a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
+++ b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
@@ -91,6 +91,7 @@ content::RenderProcessHost* WebrtcLoggingPrivateFunction::RphFromRequest(
     const api::webrtc_logging_private::RequestInfo& request,
     const std::string& security_origin,
     std::string* error) {
+  BRAVE_DISABLE_RPH_FROM_REQUEST //feature disabled in Brave
   // There are 2 ways these API functions can get called.
   //
   //  1. From an allowlisted component extension on behalf of a page with the
