diff --git a/chrome/browser/ui/tabs/tab_strip_prefs.cc b/chrome/browser/ui/tabs/tab_strip_prefs.cc
index e0b61790305ef02248318477da2b5ffacbcf7644..7e0eaf09493406c78f1c31e9c7cf8503a4937740 100644
--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -23,6 +23,7 @@ std::optional<bool> g_tab_search_trailing_tabstrip_at_startup = std::nullopt;
 namespace tabs {
 
 bool GetDefaultTabSearchRightAligned() {
+  BRAVE_GET_DEFAULT_TAB_SEARCH_RIGHT_ALIGNED
   // These platforms are all left aligned, the others should be right.
 #if BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
   return false;
