diff --git a/chrome/browser/ui/browser_window/public/browser_window_features.h b/chrome/browser/ui/browser_window/public/browser_window_features.h
index af75c2a8ff9c40ea25500f2a18c73de2b3b8f29e..024ffc1c9147a321e5f24789024c0656e320f4bc 100644
--- a/chrome/browser/ui/browser_window/public/browser_window_features.h
+++ b/chrome/browser/ui/browser_window/public/browser_window_features.h
@@ -232,7 +232,7 @@ class SkillsUiWindowController;
 class BrowserWindowFeatures {
  public:
   BrowserWindowFeatures();
-  ~BrowserWindowFeatures();
+  virtual ~BrowserWindowFeatures();
 
   BrowserWindowFeatures(const BrowserWindowFeatures&) = delete;
   BrowserWindowFeatures& operator=(const BrowserWindowFeatures&) = delete;
