diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index c775ae9ce2b58d010f9c8f9f8033f3e7e40627de..07be443cd42cd6849a01e8bd14ef7190d6c4890b 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1717,6 +1717,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
       false);
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2209,6 +2210,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
 #endif  // BUILDFLAG(IS_ANDROID)
 
   registry->RegisterBooleanPref(prefs::kStaticStorageQuotaEnabled, false);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -2631,6 +2633,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
   profile_prefs->ClearPref(kObsoletePasswordsUseUPMLocalAndSeparateStores);
   profile_prefs->ClearPref(kObsoleteEmptyProfileStoreLoginDatabase);
   profile_prefs->ClearPref(kObsoleteUpmAutoExportCsvNeedsDeletion);
+#if 0
   base::DeleteFile(profile_path.Append(FILE_PATH_LITERAL("Login Data")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account")));
@@ -2638,6 +2641,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
       profile_path.Append(FILE_PATH_LITERAL("Login Data-journal")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account-journal")));
+#endif
 #endif  // BUILDFLAG(IS_ANDROID)
 
   // Added 09/2025.
