diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index 732a393d57685728272002b0d33d9af6ef7e8b43..5183904740d60992d248d9fc483b7786328109df 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -34,6 +34,7 @@ source_set("browser_window") {
       "//ui/base:types",
     ]
   }
+  deps += [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("create_browser_window") {
