diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index c62d6f46cf2c2e53307def983a1555f070f80a33..74788f4e22374a97bf1d3c6f9b11bcd0e9da15da 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -779,6 +779,7 @@ static_library("ui") {
     # gets componentized.
     "//chrome/browser/ui/extensions:impl",
   ]
+  import("//brave/browser/sources.gni") import("//brave/components/brave_wallet/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from + brave_chrome_browser_ui_deps allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3377,7 +3378,10 @@ static_library("ui") {
         "//chrome/updater/app/server/win:updater_legacy_idl",
       ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5521,6 +5525,7 @@ static_library("ui_public_dependencies") {
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
@@ -5533,6 +5538,7 @@ static_library("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
@@ -5673,6 +5679,7 @@ source_set("browser_navigator_params_headers") {
     "//url",
   ]
   deps = []
+  public_deps += [ "//brave/browser/ui:browser_navigator_params_headers_public_dependencies" ]
 }
 
 if (!is_android) {
