diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 915a59ba7c1081ced902160258709b019e9fc6db..8adffe7d43a8e7a3ff0a267733322a586a5a983a 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1159,6 +1159,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
