diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
index 88cc637570d056ba6d8fe941c55a1ac22d59e05c..165a12ffe986fea731dceb565b1d43bacd2d0723 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
@@ -39,7 +39,7 @@ class TabStripFlatEdgeButton;
 
 // Container for the tabstrip and the other views sharing space with it -
 // with the exception of the caption buttons.
-class HorizontalTabStripRegionView final : public TabStripRegionView {
+class HorizontalTabStripRegionView : public TabStripRegionView {
   METADATA_HEADER(HorizontalTabStripRegionView, TabStripRegionView)
 
  public:
