diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 08206105c32c819eaea08e2581fd898078a71e11..361c4660d7ef641b5cbfe41ec03c2ac12cdafccc 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -139,6 +139,7 @@ android_library("core_java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
@@ -345,6 +346,7 @@ android_resources("java_resources") {
     "//components/browser_ui/styles/android:java_resources",
     "//ui/android:ui_java_resources",
   ]
+  deps += [ "//brave/browser/ui/android/toolbar:java_resources" ]
 }
 
 robolectric_library("junit") {
