diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index f4ad662dc769521484831fc9935b734d600c36ce..ebaeba49b7dd188725078a1d10af7f86f184dbdf 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -438,7 +438,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
