diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7fa426ab4900335052a3f4d8c79f6fe3b738528d..0fbd4fc4cecb7ce87448bca6f8bce467e56f06c5 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2590,6 +2590,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
@@ -8586,6 +8587,7 @@ static_library("browser_public_dependencies") {
       "//rlz:rlz_lib",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
