diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index e94d581cfb5d372a04ac0d3c42dddad1336fb86e..ceffd28d7e9af8ae7e4fe0c9df0f054bbf746394 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -74,6 +74,7 @@
 #include "chrome/browser/on_device_translation/installer_impl.h"
 #endif  // BUILDFLAG(ENABLE_ON_DEVICE_TRANSLATION)
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -92,7 +93,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
