diff --git a/components/translate/core/common/BUILD.gn b/components/translate/core/common/BUILD.gn
index 4d94552699ffe00f9f49a515040033f3ddfe8666..777e3f9060cab2e60b2277f1fc5c2c2de0fb2319 100644
--- a/components/translate/core/common/BUILD.gn
+++ b/components/translate/core/common/BUILD.gn
@@ -30,6 +30,8 @@ static_library("common") {
     "//third_party/metrics_proto",
     "//url",
   ]
+
+  import("//brave/components/translate/core/common/sources.gni") sources += brave_components_translate_core_common_sources public_deps = brave_components_translate_core_common_public_deps
 }
 
 source_set("unit_tests") {
