diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index 63da4a979ffff63ceec1674b1bd8cc11afa82080..2b78dc35de24b6ddae706d41c166acbbdcacb48b 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -214,6 +214,7 @@ class InfoBarDelegate {
     DEV_TOOLS_REMOTE_DEBUGGING_INFOBAR_DELEGATE = 130,
     STARTUP_LAUNCH_INFOBAR_DELEGATE = 131,
     AUTOFILL_AI_SAVE_ENTITY_INFOBAR_DELEGATE_IOS = 132,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
   // LINT.ThenChange(//tools/metrics/histograms/metadata/browser/enums.xml:InfoBarIdentifier)
 
