diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index 3f9c849bece239513271e19cc8f7f358ef9d45e3..10c7948ef6b84c56b188c1a2c9718635f5c1a85c 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -746,6 +746,7 @@ base::DictValue GpuMessageHandler::GetClientInfo() {
            std::string("Skia/" STRINGIZE(SK_MILESTONE) " " SKIA_COMMIT_HASH));
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
+  BRAVE_GPU_MESSAGE_HANDLER_ON_REQUEST_CLIENT_INFO
   return dict;
 }
 
