diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index c20ef31cf67546758f43bf4a62aa5b409af577b5..d524b7f45564cbbbe1df3bd1a4b4fba4569a08ae 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -63,6 +63,7 @@ source_set("common") {
   # must not link to this because it will duplicate the code in the component
   # build.
   visibility = [ "//content/*" ]
+  visibility += [ "//brave/content/*" ]
 
   sources = [
     "background_fetch/background_fetch_types.cc",
