diff --git a/sandbox/policy/win/sandbox_win.cc b/sandbox/policy/win/sandbox_win.cc
index 8497f239cf6039dea0f2e5cfad274312d9144d39..190b86d568d208eb1d5bc05f8edad24614258471 100644
--- a/sandbox/policy/win/sandbox_win.cc
+++ b/sandbox/policy/win/sandbox_win.cc
@@ -622,6 +622,7 @@ ResultCode LaunchWithoutSandbox(
   base::LaunchOptions options;
   options.handles_to_inherit = handles_to_inherit;
   options.feedback_cursor_off = true;
+  BRAVE_START_SANDBOXED_PROCESS
   // Network process runs in a job even when unsandboxed. This is to ensure it
   // does not outlive the browser, which could happen if there is a lot of I/O
   // on process shutdown, in which case TerminateProcess can fail. See
