diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index f53eebc1168c0eb56d073219af952635ceaa157b..ac09efc2e4953c2bdedf8dd249ceccd3812b7b4b 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -838,6 +838,11 @@
       public: true,
       status: "stable",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
     {
       name: "BrowserInitiatedAutomaticPictureInPicture",
       public: true,
@@ -2551,6 +2556,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2565,6 +2571,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3630,6 +3637,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
